include/nlohmann/detail/output/binary_writer.hpp

```diff
@@ -177,8 +177,27 @@ class binary_writer
 
             case value_t::number_float:
             {
-                oa->write_character(get_cbor_float_prefix(j.m_value.number_float));
-                write_number(j.m_value.number_float);
+		double valueD = j.m_value.number_float;
```

Comment:
Why make a copy here?

Reply:
Because valueD will be reused in several times

> It seems that there are other CBOR tests failing.

Yes, some other CBOR tests failed, I am checking the reason.

Reply:
I have a little doubt, I ran the executable file test-cbor, the result seems ok, no problem like those shown in CI. 
P.S. it takes too much time on ctest to download test data :-)

Reply:
I don't understand: Downloading the test data takes 8 seconds on AppVeyor and 4 seconds on Travis.